Sources/TokamakCore/Props/Font.swift

```diff
@@ -0,0 +1,167 @@
+// Copyright 2018-2020 Tokamak contributors
```

Comment:
**suggestion:** It's probably time to rename the parent directory from `Props` to `Modifiers`.

Reply:
I feel like Modifiers isn't quite the right description. `Font`, `Color`, etc. are not modifiers themselves, but instead are values used in the modifiers. SwiftUI describes both of these as late-binding tokens, as they're resolved just before being used. We don't "resolve" them like SwiftUI does to `UIFont`/`UIColor` respectively, but they are still resolved to CSS when the view is built... Maybe call the folder `Tokens`?

Reply:
`Tokens` makes sense, thanks for clarifying.